Doc/library/zipfile.rst

```diff
@@ -183,6 +184,9 @@ ZipFile Objects
       Previously, a plain :exc:`RuntimeError` was raised for unrecognized
       compression values.
 
+   .. versionchanged:: 3.6.2
```

Comment:
Thank you, it was just my misunderstanding on how the upcoming release will be cut, I assumed it's feed by the `3.6` branch.